app/assets/stylesheets/govuk_publishing_components/components/_phase-banner.scss

```diff
@@ -0,0 +1,7 @@
+@import "mixins/touch-friendly-links";
+@import "design-patterns/alpha-beta";
```

Comment:
We might want to look into this in a bit more depth as this file seems to be pulling in a lot of stuff - https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/design-patterns/_alpha-beta.scss#L1-L4. Will that cause duplication in the CSS? Is all of this needed?

Reply:
Commenting here because it's still relevant in https://github.com/alphagov/govuk_publishing_components/pull/333. GOV.UK Frontend toolkit doesn't include concrete classes, only mixins. This means that there won't be any extra CSS added because of these imports.